src/dev/i18n/extract_i18n_call_messages.test.js

```diff
@@ -0,0 +1,123 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { parse } from '@babel/parser';
+import { isCallExpression } from '@babel/types';
+
+import { extractI18nCallMessages } from './extract_i18n_call_messages';
+import { traverseNodes } from './utils';
+
+const i18nCallMessageSource = `
+i18n('message-id-1', { defaultMessage: 'Default message 1', context: 'Message context 1' });
+`;
+
+const translateCallMessageSource = `
+i18n.translate('message-id-2', { defaultMessage: 'Default message 2', context: 'Message context 2' });
+`;
+
+describe('extractI18nCallMessages', () => {
+  it('extracts "i18n" and "i18n.translate" functions call message', () => {
+    for (const node of traverseNodes(parse(i18nCallMessageSource).program.body)) {
+      if (isCallExpression(node)) {
+        const actual = extractI18nCallMessages(node);
+        const expected = [
+          'message-id-1',
+          {
+            message: 'Default message 1',
+            context: 'Message context 1',
+          },
+        ];
+
+        expect(actual).toEqual(expected);
```

Comment:
__nit:__ same note about `expect` inside of `if` here as well (and in all other tests in this file).